mpicbg_/src/main/java/Transform_Affine.java

```diff
@@ -61,4 +63,10 @@ final protected void updateHandles( final int x, final int y )
 		fq[ 0 ] = x;
 		fq[ 1 ] = y;
 	}
+
```

Comment:
`@Override` requires that you depend on the new version of mpicbg instead of on the current release.  Do you use m2e with project resolution?  That shuold fix it.